pkg/app/web/src/components/applications-page/add-application-drawer/index.tsx

```diff
@@ -60,6 +64,11 @@ export const AddApplicationDrawer: FC<AddApplicationDrawerProps> = memo(
       }
     }, [onClose, formik]);
 
+    const [showConfirmToAddFromGit, setShowConfirmToAddFromGit] = useState(false);
+    const handleAddFromGit = useCallback(() => {
+      setShowConfirmToAddFromGit(true);
+    }, []);
```

Comment:
nits, we may not need to use this `useCallback`

Reply:
I see. No input is given then it has no sense 👍